docs/tendermint-core/reactors/block-sync/bcv0/implementation.md

```diff
@@ -0,0 +1,47 @@
+---
+order: 3
+---
+
+# Implementation
```

Comment:
there are many things in this file that ought to be marked as code/pre with backtics